rocky/reports/forms.py

```diff
@@ -47,25 +57,32 @@ class ReportScheduleStartDateForm(BaseRockyForm):
         input_formats=["%Y-%m-%d"],
     )
 
-
-class ReportRecurrenceChoiceForm(BaseRockyForm):
-    choose_recurrence = forms.ChoiceField(
-        label="",
-        required=False,
-        widget=forms.RadioSelect(attrs={"class": "submit-on-click"}),
-        choices=(("once", _("No, just once")), ("repeat", _("Yes, repeat"))),
-        initial="once",
+    start_time = forms.TimeField(
+        label=_("Start time (UTC)"),
+        widget=forms.TimeInput(format="%H:%M", attrs={"form": "generate_report"}),
+        initial=lambda: datetime.now(tz=timezone.utc).time(),
+        required=True,
+        input_formats=["%H:%M"],
     )
 
-
-class ReportScheduleRecurrenceForm(BaseRockyForm):
     recurrence = forms.ChoiceField(
         label=_("Recurrence"),
         required=True,
         widget=forms.Select(attrs={"form": "generate_report"}),
         choices=[("daily", _("Daily")), ("weekly", _("Weekly")), ("monthly", _("Monthly")), ("yearly", _("Yearly"))],
     )
 
+    def clean(self):
+        cleaned_data = super().clean()
+        start_date = cleaned_data.get("start_date")
+        start_time = cleaned_data.get("start_time")
+
+        if start_date and start_time:
+            start_datetime = datetime.combine(start_date, start_time)
```

Comment:
Don't need to add this if using `models.DateTimeField` to have both Date and time together in the widget.

Reply:
See other comment. We'll keep those separated.